e2e/etcd_release_upgrade_test.go

```diff
@@ -37,7 +35,11 @@ func TestReleaseUpgrade(t *testing.T) {
 	copiedCfg := configNoTLS
 	copiedCfg.execPath = lastReleaseBinary
 	copiedCfg.snapCount = 3
-	copiedCfg.baseScheme = "unix" // to avoid port conflict
+
```

Comment:
we should not change this test. we should add a new test called TestReleaseUpgradeWithSnapshot.